src/adapter/sources.ts

```diff
@@ -438,7 +438,7 @@ export class SourceContainer {
     this._sourceByReference.set(source.sourceReference(), source);
     if (source._compiledToSourceUrl)
       this._sourceMapSourcesByUrl.set(source._url, source);
-    this._sourceByAbsolutePath.set(source._absolutePath, source);
+    this._sourceByAbsolutePath.set(utils.lowerCaseInsensitivePath(source._absolutePath), source);
```

Comment:
Does it make sense to create a specific "file map" class for this, so we don't have to remember to call `utils.lowerCaseInsensitivePath` every time we access this map?

Reply:
Done :)